src/common/date-range/legend/DateRangeLegend.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import injectT from '../../../../app/i18n/injectT';
+
+const DateRangeLegend = (props) => {
+  const { t } = props;
+  return (
+    <div className="app-DateRangeLegend">
+      <div className="legend-item">
+        <div className="color-box color-box__available">01</div>
```

Comment:
Prefixing these classNames with `app-DateRangeLegend` would reduce the possibility of conflicting with other classNames.

Reply:
Done :+1: